components/survey/Card/index.jsx

```diff
@@ -0,0 +1,47 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import Image from 'next/image';
+
+import forwardButton from '@/public/angle-right.svg';
```

Comment:
```suggestion
import forwardIcon from '@/public/angle-right.svg';
```

or why not even:

```suggestion
import AngleRightIcon from '@/public/angle-right.svg';
```
(if you like the name "forward", maybe we can rename the svg to "forward-icon.svg" too 🤔 )